basis_universal.lua

```diff
@@ -0,0 +1,610 @@
+
+  -- common configuration settings
+
+  dofile (_BUILD_DIR .. "/static_android_armv7_release.lua")
+
+  -- project specific configuration settings
+
+  -- configuration { "android_armv7_release" }
+
+  -- -------------------------------------------------------------
+  -- configuration { "android_x86_debug" }
+  -- -------------------------------------------------------------
+
+  -- common configuration settings
+
+  dofile (_BUILD_DIR .. "/static_android_x86_debug.lua")
+
+  -- project specific configuration settings
+
+  -- configuration { "android_x86_debug" }
+
+  -- -------------------------------------------------------------
+  -- configuration { "android_x86_release" }
+  -- -------------------------------------------------------------
+
+  -- common configuration settings
+
+  dofile (_BUILD_DIR .. "/static_android_x86_release.lua")
+
+  -- project specific configuration settings
+
+  -- configuration { "android_x86_release" }
+
+  -- -------------------------------------------------------------
+  -- configuration { "android_arm64_debug" }
+  -- -------------------------------------------------------------
+
+  -- common configuration settings
+
+  dofile (_BUILD_DIR .. "/static_android_arm64_debug.lua")
+
+  -- project specific configuration settings
+
+  -- configuration { "android_arm64_debug" }
+
+  -- -------------------------------------------------------------
+  -- configuration { "android_arm64_release" }
+  -- -------------------------------------------------------------
+
+  -- common configuration settings
+
+  dofile (_BUILD_DIR .. "/static_android_arm64_release.lua")
+
+  -- project specific configuration settings
+
+  -- configuration { "android_arm64_release" }
+
+  -- -------------------------------------------------------------
+  -- configuration { "android_x64_debug" }
+  -- -------------------------------------------------------------
+
+  -- common configuration settings
+
+  dofile (_BUILD_DIR .. "/static_android_x64_debug.lua")
+
+  -- project specific configuration settings
+
+  -- configuration { "android_x64_debug" }
+
+  -- -------------------------------------------------------------
+  -- configuration { "android_x64_release" }
+  -- -------------------------------------------------------------
+
+  -- common configuration settings
+
+  dofile (_BUILD_DIR .. "/static_android_x64_release.lua")
+
+  -- project specific configuration settings
+
+  -- configuration { "android_x64_release" }
+
+  -- -------------------------------------------------------------
+end
+
+if (_PLATFORM_WINUWP) then
+  -- -------------------------------------------------------------
+  -- configuration { "windows" }
+  -- -------------------------------------------------------------
+
+  -- common configuration settings
+
+  dofile (_BUILD_DIR .. "/static_winuwp.lua")
+
+  -- project specific configuration settings
+
+  configuration { "windows" }
+
+    buildoptions {
+      "/sdl",
```

Comment:
These are only needed when using pplx and can be removed.

Reply:
The entire `buildoptions` block can be removed? Or just the options below "/sdl"?

Reply:
I would remove all of them. The sdl is just warnings which we don't really care about in 3rdparty since we don't change the code.